hydra-web/src/js/modules/host.js

```diff
@@ -224,8 +224,8 @@ function(
 					alertify.error("Fatal warning for failing " + uuids + ": " + data.fatal + "; fail aborted");
 					return;
 				}
-				var msg = "Are you sure you want to fail " + uuids + "?\n";
-				msg += "After failing, cluster will go from " + util.formatPercent(data.prefail) + "% disk used to " + util.formatPercent(data.postfail) + "%.\n";	
+				var msg = "<span class=\"bold-red\">Do you seriously want to fail (file system dead) " + uuids + "?</span><br/>";
+				msg += "(After failing, cluster will go from " + util.formatPercent(data.prefail) + "% disk used to " + util.formatPercent(data.postfail) + "%)";
```

Comment:
I know you made two pull requests, but I'm going to comment on all issues just in this one since it has both commits.

This change here will result in both failing fs-dead and fs-okay to both say "Do you seriously want to fail (file system dead)" and that's not correct. Please make use of the `deadFs` variable passed into this function to fix this.